ml-agents/mlagents/trainers/trainer_controller.py

```diff
@@ -173,7 +201,9 @@ def initialize_trainers(self, trainer_config: Dict[str, Dict[str, str]]):
                         brain_name
                     ].min_lesson_length
                     if self.meta_curriculum
-                    else 0,
+                    else self.lesson_controller.min_lesson_length
```

Comment:
The logic for this was debatably too complicated before, it definitely is now. Please extract it to a local variable or a helper method.